templates/compose/pterodactyl-with-wings.yaml

```
@@ -101,41 +106,125 @@ services:
       - MAIL_USERNAME=$MAIL_USERNAME
       - MAIL_PASSWORD=$MAIL_PASSWORD
       - MAIL_ENCRYPTION=$MAIL_ENCRYPTION
-
   wings:
-    image: "ghcr.io/pterodactyl/wings:latest"
+    image: 'ghcr.io/pterodactyl/wings:latest'
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

---

[v1.11.13](https://github.com/orgs/pterodactyl/packages/container/wings/213438525?tag=v1.11.13)
```suggestion
    image: 'ghcr.io/pterodactyl/wings:v1.11.13'
```